src/main/java/com/tesco/mewbase/server/ServerOptions.java

```
@@ -20,6 +21,8 @@
     private String docsDir = DEFAULT_DOCS_DIR;
     private String[] binders;
 
+    private MewbaseAuthProvider authProvider;
```

I think we should probably default to the DummyAuthProvider (i.e. no auth)